test/powershell/Modules/CimCmdlets/CimInstance.Tests.ps1

```diff
@@ -13,27 +13,27 @@ Try {
             # we can't use equals here as on windows cimclassname
             # is win32_computersystem, but that's not likely to be the
             # case on non-Windows systems
-            $instance.cimClass.CimClassName | should match _computersystem
+            $instance.cimClass.CimClassName | Should -Match _computersystem
         }
         It "Property access should be case insensitive" {
             foreach($property in $instance.psobject.properties.name) {
                 $pUpper = $property.ToUpper()
                 $pLower = $property.ToLower()
                 [string]$pLowerValue = $pinstance.$pLower -join ","
                 [string]$pUpperValue = $pinstance.$pUpper -join ","
-                $pLowerValue | should be $pUpperValue
+                $pLowerValue | Should -Be $pUpperValue
```

Comment:
The `-BeExactly` should rather be used in this scenario since the documentation [here](https://github.com/pester/Pester/wiki/Should) states that the `-Be` operator is not case sensitive. The test compares whether property access is case insensitive but there is also value in checking that the actual value is also exactly the same